styleguide/src/Icons/icons-path/Check.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export const Check = () => (
+  <path
```

Comment:
Sabe se o viewport desse é igual aos demais?
![image](https://user-images.githubusercontent.com/3163867/134226998-afbf7e1c-94c7-4975-abc0-5e0634da1dfd.png)

Para que esta um pouco desalinhado.
Se for, pode adicionar mais um IF (que inclusive não acho uma boa solução a implementada, mas não tive tempo de pensar em algo mais bonito, aceitamos PRs kkkkk)
https://github.com/lojaintegrada/admin-components/blob/main/styleguide/src/Icons/Icon.tsx#L27

Reply:
Tem outros também que não ficaram bons, exemplo: plus circle. Vou revisar